sitemedia/scss/components/_shelfmark.scss

```diff
@@ -2,8 +2,9 @@
 // Styling rules for document shelfmarks.
 // -----------------------------------------------------------------------------
 
-.shelfmark span {
-    // Never break in the middle of a shelfmark
+.shelfmark span,
+time span {
```

Comment:
The component name isn't accurate anymore, but the styling is the exact same; suggestions?